tensorflow/tools/api/golden/v2/tensorflow.summary.pbtxt

```diff
@@ -10,7 +10,7 @@ tf_module {
   }
   member_method {
     name: "audio"
-    argspec: "args=[\'name\', \'data\', \'sample_rate\', \'step\', \'max_outputs\', \'encoding\', \'description\'], varargs=None, keywords=None, defaults=[\'None\', \'3\', \'None\', \'None\'], "
```

Comment:
Why these changes to the summary file? Are you out of sync with HEAD?

Reply:
I synced my fork master and this PR related branch with remote master, then rerun 
```
$ bazel build tensorflow/tools/api/tests:api_compatibility_test
$ bazel-bin/tensorflow/tools/api/tests/api_compatibility_test \
          --update_goldens True
```
it shows the same changes in the summary file. I am not sure what leads to this irrelevant change.
But I can change it back manually, and push the old summary file. do you think it is a good idea?

Reply:
Ok, no need to worry about this, then.